src/JsonSerializer.php

```diff
@@ -1,31 +1,47 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Yiisoft\Serializer;
 
+use function json_decode;
+use function json_encode;
+
 /**
  * JsonSerializer serializes data in JSON format.
  */
 final class JsonSerializer implements SerializerInterface
 {
-    private $options;
+    /**
+     * @param int integer the encoding options.
+     * @see http://www.php.net/manual/en/function.json-encode.php
+     */
+    private int $options;
 
     /**
-     * JsonSerializer constructor.
-     *
-     * @param int $options integer the encoding options. For more details please refer to
-     * <http://www.php.net/manual/en/function.json-encode.php>.
-     * Default is `JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE`.
+     * @param int $options integer the encoding options.
+     * @see http://www.php.net/manual/en/function.json-encode.php
      */
     public function __construct(int $options = JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE)
     {
         $this->options = $options;
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * @see json_encode()
```

Comment:
This phpdoc doesn't add anything since `@inheritDoc` is default IDE behavior and it's obvious to check method implementation that is `json_encode`.